stories/assets/js/smoothbgchange.js

```diff
@@ -3,16 +3,17 @@ $(window).scroll(() => {
   var $body = $('body');
   var $container = $('.heading-big.inviewport');
 
-  var scroll = $window.scrollTop() + ($window.height() / 3);
+  var scroll = $window.scrollTop() + ($window.height() / 2);
```

Comment:
@neelesh-tippe please create all the jquery variables as starting with '$' like it should be $scroll